### ISSUE TITLE
Battle to chose starting game due to Wand --(crazy fast scrolling!)...

### ISSUE THREAD

**Issue by OrionUk** [LibreVR/Revive#980]
Oculus & Revivie..Will you PLEASE...(for the love of God!) Change the way game selection is made! I BEG you, unhook scrolling from the wand and just use the touch pad to scroll!...having them both just confuses the issue/selection process, and as you can see is a total nightmare!!!!....JUST touch pad please? Want makes things WAY too much hassle/effort... I want to fight Zombies, not menus!

Please see video posted here: https://youtu.be/wEoIpDCxwmc
(Once it has uploaded!) Thank you :)

**Comment by Armada651** [comment]
Your video never completed uploading.

**Comment by OrionUk** [comment]
Hhhmmm strange.....

Does this work?...works my end...
https://www.youtube.com/watch?v=C2O_JW2tPGE&feature=youtu.be

**Comment by Armada651** [comment]
You need to describe your issue better, to me it just looks like you're scrolling using the touchpad as normal.

You should use the trigger to select the game you want to play after scrolling to the correct page.

**Comment by Armada651** [comment]
To clarify, there are two ways to scroll the page: The touchpad and by keeping the trigger pressed. Maybe you're accidentally keeping the trigger pressed?

**Comment by OrionUk** [comment]
Hi, thank you for replying....
Well basically...my issue is that the wand is 'auto scrolling' (with out me touching the trigger! or touch pad) at a very fast rate, and if i move the wand tip down it 'shoots past' what i want/if i move the wand tip up..it shoots past the program i want also..i can NOT get the scrolling to stop due to the way the wand is making the scroll for going either up or down at very fast rate :( If i try to get the wand to stop on a program it just shoots past it!...as you see in the video, i was trying to chose luckys tale...after uncontrollable scrolling..it then chose they play room :(

**Comment by Armada651** [comment]
That's not supposed to happen, but I haven't heard anyone with a Vive encountering that problem. Are you sure your wands are working correctly?

**Comment by OrionUk** [comment]
Yes, they work flawlessly with all my games/programs...just this situation with Revivie i am sorry to say.
Are there any log/output files i can gather that might help?

**Comment by Armada651** [comment]
For now you can use the Steam tab to start games, it shows the same applications as the Revive dashboard. That should work as a temporary workaround.

**Comment by OrionUk** [comment]
ah great, thanks :)
